### PR TITLE
Fix multiprocess and bugs

### PR DESCRIPTION
**Pull request by bangxiangyong**
Quite urgent fixes i'd say on two minor things: 
1) using Process for Dashboard on osbrain backend
2) dashboard monitor agent key issue

### CODE REVIEW

**Comment by bangxiangyong** [comment]
This is strange, with my desktop, i have managed to pass all the tests for `test_memory_monitor_agent.py` which circleci failed on.

File: agentMET4FOF/dashboard/Dashboard.py [hunk 1]
```diff
@@ -163,7 +163,8 @@ def is_port_available(self, ip_addr, _port):
             return True
 
 
-class AgentDashboardProcess(AgentDashboard, Process):
+
+class AgentDashboardProcess(AgentDashboard, pathos.helpers.mp.Process):
```
Comment (BjoernLudwigPTB) [inline agentMET4FOF/dashboard/Dashboard.py:167]: I am currently looking through your changes and will finish this tomorrow morning. Can you shortly summarize what your intention was with using `pathos.helpers.mp.Process`?

**Review by BjoernLudwigPTB — COMMENTED** [review]
Nice that you finally touched this easy fix for the `sender_agent` issue, thanks! I am currently looking through your changes and will finish this tomorrow morning. Can you shortly summarize what your intention was with using `pathos.helpers.mp.Process`?

**Comment by BjoernLudwigPTB** [comment]
> This is strange, with my desktop, i have managed to pass all the tests for `test_memory_monitor_agent.py` which circleci failed on.

Sometimes something strange happens on these failed tests, which appears to be kind of random. Usually reexecuting the same tests on the same commits works.

**Comment by BjoernLudwigPTB** [comment]
Just found your comment in #176 about fixing the `Process` issues for distributed systems with this. Awesome!

**Pull request merged**